client/src/services/LoginService.js

```diff
@@ -1,50 +1,49 @@
+import ErrorMessage from 'models/ErrorMessages';
+
 export default class LoginService {
   static login = (data) => {
+
     const formData = new FormData();
 
     formData.append('username', data.username);
     formData.append('password', data.password);
 
-
-    // Configure POST request options
-    const fetchOptions = {
+    const postRequestOptions = {
       method: 'POST',
       body: formData,
     };
 
+
     return new Promise(async (resolve, reject) => {
       try {
-        // Sends the POST Request
-        // Catch network errors and thow them to the catch block
+
+        /* Sends the POST Request */
```

Comment:
Just using a different commenting syntax to differentiate between comments and "sections". I.e next "section" is the error checking.